tailcall-fixtures/fixtures/configs/user-posts.graphql

```diff
@@ -17,7 +17,7 @@ type User {
   username: String!
   email: String!
   phone: String
-  website: String @expr(body: "/users/website/{{.value.username}}")
+  websiteExpr: String @expr(body: "/users/website/{{.value.username}}")
```

Comment:
```suggestion
  website: String @expr(body: "www.{{.value.username}}.com")
```

Reply:
no, website is already defined in jsonplaceholder. That's the issue I was talking on the call about store/parent value priority